PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -50,7 +50,7 @@ generic_string commafyInt(size_t n)
 	return ss.str();
 }
 
-std::string getFileContent(const TCHAR *file2read)
+std::string getFileContent(const wchar_t *file2read)
 {
 	if (!::PathFileExists(file2read))

```

Comment:
PathFileExistsW

@donho 
I will not continue in this review, there are so many problems of this type.
E.g. the nearby line 63 should be: `FILE *fp = _wfopen(file2read, L"rb");`